src/wp-admin/admin-header.php

```diff
@@ -232,11 +231,11 @@
 $admin_body_classes = ltrim( $admin_body_classes . ' ' . $admin_body_class );
 ?>
 <body class="wp-admin wp-core-ui no-js <?php echo $admin_body_classes; ?>">
-<script type="text/javascript">
-	document.body.className = document.body.className.replace('no-js','js');
-</script>
-
 <?php
+
+$js = 'document.body.className = document.body.className.replace("no-js","js");';	
+inline_js( $js );
```

Comment:
See above, this is short enough to not have a variable. Would be great if you could check for that throughout the rest of the PR.